openmetadata-service/src/main/java/org/openmetadata/service/resources/apps/AppResource.java

```diff
@@ -756,6 +764,13 @@ public Response delete(
       throw new IllegalArgumentException(
           CatalogExceptionMessage.systemEntityDeleteNotAllowed(app.getName(), "SystemApp"));
     }
+
```

Comment:
remove this

Reply:
handle it in app specific function overriden for the app

Reply:
Noted.